jre8-alpine/Dockerfile

```diff
@@ -1,19 +1,32 @@
-FROM openjdk:8-jre-alpine
+FROM ubuntu:18.04
+
 LABEL Azure App Services Container Images <appsvc-images@microsoft.com>
 ADD default.jar /tmp/webapps/default.jar
 ADD init_container.sh /bin/init_container.sh
 ADD sshd_config /etc/ssh/
 ENV JAVA_OPTS=""
 ENV _JAVA_OPTIONS -Djava.net.preferIPv4Stack=true
 
+#
+# Pull Zulu OpenJDK binaries from official repository:
+#
+
+RUN apt-get -qq update
+RUN apt-get -y install gnupg software-properties-common
+
+RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0x219BD9C9 && \
```

Comment:
>0x219BD9C9 [](start = 70, length = 10)

is this the key for Commercial bits for Azure deployments or still community edition? It will make more sense to make this explicit in the comment above